lib/wallaroo/core/sink/connector_sink/connector_sink.pony

```diff
@@ -662,7 +704,13 @@ actor ConnectorSink is Sink
 
   be prepare_for_rollback() =>
     @ll(_conn_debug, "Prepare for checkpoint rollback at ConnectorSink %s".cstring(), _sink_id.string().cstring())
-    // Don't call _use_normal_processor() here
+    match _phase
```

Comment:
We should delegate to the phase and never match on it.  The correct phase would then call `_use_normal_processor()` on this. The default method body can be `None`.

Reply:
Fixed by 21dfe5a4f